app/components/pages/Live.vue.ts

```diff
@@ -26,16 +27,33 @@ export default class Live extends Vue {
   @Inject() userService: UserService;
   @Inject() customizationService: CustomizationService;
   @Inject() i18nService: I18nService;
+  @Inject() navigationService: NavigationService;
 
   $refs: {
     webview: Electron.WebviewTag;
   };
-  
+
   enablePreviewTooltip = $t('Enable the preview stream');
   disablePreviewTooltip = $t('Disable the preview stream, can help with CPU');
 
   mounted() {
     this.i18nService.setWebviewLocale(this.$refs.webview);
+
+    this.$refs.webview.addEventListener('new-window', e => {
+      const match = e.url.match(/dashboard\/([^\/^\?]*)/);
+
+      if (match && match[1] === 'recent-events') {
+        this.popout();
+      } else if (match) {
```

Comment:
I'm a bit curious in which case we should redirect here?

Reply:
There are a couple buttons in the recent events feed that link to the spin wheel and credits widget settings.  Before this change they have just never worked.  I saw an opportunity to make them start doing the correct thing pretty easily in the context of this larger change.